core/java/iesi-core/src/main/java/io/metadew/iesi/common/crypto/JavaKeystore.java

```diff
@@ -0,0 +1,39 @@
+package io.metadew.iesi.common.crypto;
+
+import io.metadew.iesi.common.configuration.Configuration;
+import org.apache.commons.io.FileUtils;
+
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableEntryException;
+import java.security.cert.CertificateException;
+import java.security.spec.InvalidKeySpecException;
+
+//keytool -importpass -storetype pkcs12 -alias mypass -keystore Desktop/myks.p12
+public class JavaKeystore {
+
+    public String loadKey(String keyStorePassword, String keystoreLocation) throws KeyStoreException, NoSuchAlgorithmException, UnrecoverableEntryException, InvalidKeySpecException, IOException, CertificateException {
+        Configuration configuration = Configuration.getInstance();
+        KeyStore ks = KeyStore.getInstance("PKCS12");
+        String alias = configuration.getProperty("iesi.security.encryption.alias").get().toString();
```

Comment:
use Configuration.getInstance().getMandatoryProperty("iesi.security.encryption.alias") instead of getProperty().get()